src/Uno.UWP/Data/Pdf/PdfPage.Android.cs

```diff
@@ -0,0 +1,146 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Android.Graphics;
+using Android.Graphics.Pdf;
+using Uno;
+using Windows.Foundation;
+using Windows.Storage.Streams;
+using Windows.UI;
+using Color = Windows.UI.Color;
+using Rect = Windows.Foundation.Rect;
+
+namespace Windows.Data.Pdf;
+
+public partial class PdfPage : IDisposable
+{
+	private readonly PdfRenderer.Page _pdfPage;
+	// It is an empirical value found by rendering through RenderTargetBitmap
+	// in png and comparing the dimensions of the images thus obtained.
+	private const float pointToPixelFactor = 96.00000000f / 72.00000000f;
+
+	internal PdfPage(PdfRenderer.Page pdfPage)
+	{
+		_pdfPage = pdfPage ?? throw new ArgumentNullException(nameof(pdfPage));
+	}
+
+	[NotImplemented("__ANDROID__")]
+	public PdfPageDimensions Dimensions { get; } = new PdfPageDimensions();
+
+	public uint Index => (uint)_pdfPage.Index;
+
+	public Size Size => new Size(Math.Ceiling(_pdfPage.Width * pointToPixelFactor) + double.Epsilon, Math.Ceiling(_pdfPage.Height * pointToPixelFactor) + double.Epsilon);
+
+	public IAsyncAction RenderToStreamAsync(IRandomAccessStream outputStream)
+	{
+		if (outputStream is null)
+		{
+			throw new ArgumentNullException(nameof(outputStream));
+		}
+
+		var options = new PdfPageRenderOptions
+		{
+			DestinationWidth = (uint)Size.Width,
+			DestinationHeight = (uint)Size.Height
+		};
```

Comment:
Avoids re-calculating the size.

```suggestion
		var size = Size;
		var options = new PdfPageRenderOptions
		{
			DestinationWidth = (uint)size.Width,
			DestinationHeight = (uint)size.Height
		};
```